AIDevGallery/Samples/WCRAPIs/BackgroundRemover.xaml

```diff
@@ -9,100 +9,72 @@
     xmlns:samples="using:AIDevGallery.Samples"
     xmlns:shared="using:AIDevGallery.Samples.SharedCode"
     mc:Ignorable="d">
-    <Grid ColumnSpacing="24" RowSpacing="12">
+    <Grid RowSpacing="12">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="*" />
-        </Grid.ColumnDefinitions>
+        <Viewbox 
+            x:Name="ImageViewbox" 
+            Grid.Row="0"
+            Stretch="Uniform"
+            MaxWidth="600"
+            MaxHeight="400">
+            <Border 
```

Comment:
Do we need the border? Can we move its properties to the grid containing the image instead? Or vice-versa